sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -37,6 +39,9 @@ public final class SentryAndroid {
   private static final String FRAGMENT_CLASS_NAME =
       "androidx.fragment.app.FragmentManager$FragmentLifecycleCallbacks";
 
+  private static final @NotNull FullyDrawnReporter fullyDrawnReporter =
+      FullyDrawnReporter.getInstance();
```

Comment:
I don't really like this one. What would be the best way to use this `FullyDrawnReporter`?
We cannot add it to the hub, as it takes the Activity as parameter and Java doesn't have Android dependencies.
Maybe we can add the `FullyDrawnReporter` to the `SentryAndroidOptions`?